crs_motion_planning/src/path_planning_utils.cpp

```diff
@@ -449,6 +540,25 @@ bool crsMotionPlanner::generateOMPLSeed(const tesseract_motion_planners::JointWa
   }
 
   std::cout << "OMPL SEED GENERATED" << std::endl;
+
+  if (ompl_planner_response.joint_trajectory.trajectory.rows() < 5)
```

Comment:
Is it checking that it has less than 5 trajectory points  or DOF's ?

Reply:
It's checking that there are at least 5 trajectory points so that jerk smoothing can be performed and then interpolating extra points if there are less than 5